testdata/dnn/download_models.py

```diff
@@ -1049,6 +1049,36 @@ def get_confirm_token(response):  # in case of large files
         member='PP_OCRv3_DB_text_det/PP_OCRv3_DB_text_det.onnx',
         sha='f541f0b448561c7ad919ba9fffa72ff105062934',
         filename='onnx/models/PP_OCRv3_DB_text_det.onnx'),
+    Model(
+        name='YOLOv5n (ONNX)',
+        url='https://github.com/ultralytics/yolov5/releases/download/v7.0/yolov5n.onnx',
+        sha='90aa4b5f8e1ba19166f214f3e4258e5d1d05070b',
+        filename='yolov5n.onnx'),
+    Model(
+        name='YOLOv8n (ONNX)',
+        url='https://github.com/WanliZhong/opencv_extra_data_backup/blob/main/perf_test_data/models/yolov8n.onnx',
```

Comment:
Please test changes locally:

```
Model <YOLOv8n (ONNX)>
  expect fc4ffdb330de7fe1c8e5f45e71fe3173debb288b
  verify [Errno 2] No such file or directory: 'yolov8n.onnx'
  hash check failed - downloading
  get https://github.com/WanliZhong/opencv_extra_data_backup/blob/main/perf_test_data/models/yolov8n.onnx
  200 [0.0017652511596679688 Mb]
  progress > done
 file yolov8n.onnx
  expect fc4ffdb330de7fe1c8e5f45e71fe3173debb288b
  actual 1763c0561b3d0ec1eba1913fbef5d981ec4c9ecd
  renaming invalid file to yolov8n.onnx.invalid
```

This URL returns JSON metadata output.

`?raw=true` is required for GitHub repo URLs.